hooks-admin.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+// Filter export formats array
+function pb_mpdf_add_to_formats( $formats ) {
+		unset( $formats['standard']['pdf'] );
```

Comment:
Addressed this—I'm inserting mPDF via splice now (see https://github.com/BCcampus/pressbooks-mpdf/pull/17/commits/2d4700213c63148eafe7dac73081ac408463e6ad).